sklearn/model_selection/tests/test_search.py

```diff
@@ -332,6 +333,38 @@ def test_grid_search_groups():
         gs.fit(X, y)
 
 
+def test_future_warnings():
+    # Test that warnings are raised. Will be removed in 0.22
+
+    X = np.arange(100).reshape(10, 10)
+    y = np.array([0] * 5 + [1] * 5)
+
+    clf = LinearSVC(random_state=0)
+    array = np.ones(10)
+    grid = {'C': array}
+    searchcv = [GridSearchCV, RandomizedSearchCV]
+    estimators = [LinearSVC(random_state=0), LinearSVC(random_state=0)]
+    array = np.ones(10)
+    grid = {'C': array}
+    msg = ("Computing training scores may affect performance "
+           "significantly. This is the reason return_train_score will "
+           "change its default value from True (current behaviour) to "
+           "False in 0.22. Please set explicitly return_train_score to "
+           "get rid of this warning.")
+
+    for i, estimator in enumerate(estimators):
+        search = searchcv[i](clf, grid)
```

Comment:
Can't you just use estimator here? I don't see why enumerate is needed

Reply:
I guess it is necessary to loop over the GridSearchCV and RandomizedSearchCV. Could not think of an alternate form.

Reply:
Sorry, I must have not read your proposed version of the code correctly. You should be able to loop over estimators where

```
estimators = [GridSearchCV(LinearSVC(random_state=0), grid), RandomizedSearchCV(...)]
```